### PR TITLE
Make samplesheet parser tolerable to underscores in enum names

### PR DESCRIPTION
**Pull request by Donaim**
Solves #888

### CODE REVIEW

File: micall/tests/test_sample_sheet_parser.py [hunk 1]
```diff
@@ -731,7 +731,7 @@ def test_extra_commas(self):
         self.assertEqual(ss["Experiment Name"], "10-Jul-2014")
 
 
-    @unittest.expectedFailure
+    # @unittest.expectedFailure
```
Comment (donkirkby) [inline micall/tests/test_sample_sheet_parser.py:734]:
```suggestion
```
You can just remove this line, instead of commenting it out.

**Review by donkirkby — APPROVED** [review]
Looks good. Did I give you the new sample sheet that Hope created? We should confirm that this code will parse it.

**Comment by Donaim** [comment]
> Did I give you the new sample sheet that Hope created?
No, I did not get one from you yet

> We should confirm that this code will parse it.
Agree!

**Comment by Donaim** [comment]
@donkirkby This fix is independent from changes required by the new format that we got from Hope. In other words, it does not fix the issues with the new format, but that is because they are not related to the current issue that it does fix.

**Pull request merged**